src/utilities/buildASTSchema.js

```diff
@@ -466,6 +447,42 @@ export function buildASTSchema(ast: DocumentNode): GraphQLSchema {
 }
 
 /**
+ * This takes the ast of a schema document produced by the parse function in
+ * src/language/parser.js.
+ *
+ * If no schema definition is provided, then it will look for types named Query
+ * and Mutation.
+ *
+ * Given that AST it constructs a GraphQLSchema. The resulting schema
+ * has no resolve methods, so execution will use default resolvers.
+ */
+export function buildASTSchema(ast: DocumentNode): GraphQLSchema {
+  const config:any = buildASTSchemaConfig(ast);
```

Comment:
nit: if we were to keep this `any` (and I don't think we should), we'd want a space after the colon.